### DISCUSSION TITLE
2008/02/03/%E9%9B%AA%E7%9A%84%E8%AE%B0%E5%BF%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#207] [Announcements]
# 2008/02/03/%E9%9B%AA%E7%9A%84%E8%AE%B0%E5%BF%86

这几天上海下了100年来最大的一场雪，同事们都极其激动。街头、路边堆起了各式各样的雪人。又在高中同学录上有人说他那儿零下13度了，已经感觉冷的不行了，真不知道小时候是咋过来的。就是啊，这样的雪落在东北，根本不会引起任何兴奋。

https://blog.qizhen.xyz/2008/02/03/%E9%9B%AA%E7%9A%84%E8%AE%B0%E5%BF%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

liang
2008年02月22日 13:32 编辑
原来作者是齐齐哈尔的啊，我是哈尔滨的，呵呵！真是感觉很亲切

回复
li
2008年02月14日 14:42 编辑
hao  lan 好懒
 

回复
猫头鹰
2008年02月11日 16:35 编辑
我可不喜欢这雪啊。。。南方的天气总是冷得令人发指。。。
不过呢，新年快乐啊！

回复
杰
2008年02月5日 22:28 编辑
瑞雪兆丰年，
阮虾，新年快乐～

回复
arrowpig
2008年02月4日 09:32 编辑
QiZhen,以后多写些这样的文章，今年你还是去无锡过年吧。NI今天就放假了，我们还要上两天班，办公室里面人越来越少了，一年又过去了。

**Comment by ruanqizhen** [comment]
之前博客下的留言：

竹贤
2008年02月20日 08:58 编辑
冬天的回忆
四十年前，大兴安岭的冬天特别的冷，气温常常低于零下40℃。只要低于-40℃，鼻子就会有一种特殊的感觉：其实那是鼻毛冻硬了的感觉。
我到那儿第一年的冬天，有一次住进了一间平房的宿舍。下班后回到房间洗完脚，于是拖着拖鞋，端着洗脚水盆走到外面，想把水倒到房外几米外的沟里。哪知走出一两步，就发觉拖鞋开始粘地，前两步还勉强能够抬起脚，三四步后，鞋就牢牢地粘结到地上了。于是，弯下腰，用手帮忙拽拖鞋，拽下这一只，那只粘得更结实。没有办法只得放下盆，光着脚，赶紧跑进屋子。穿上棉鞋再去外面把一盆洗脚水倒在拖鞋上，才把拖鞋从地上拽出。
有一段时间我的工作是拿着长长的铁钩给蒸汽机车捅灰。那天半夜，我干了一会儿就觉得右手的虎口处特别疼痛，以为不小心弄破了。走进帐篷，摘下手套，手是好好的，既没有伤口，也没有扎刺。于是又出去，拿起铁钩，刚干几下，虎口处又剧烈疼痛。没有办法，又走进帐篷。发觉刚才还好好的虎口处的皮肤，出现了一条淡淡的白痕。一起干活的老师傅拿起我的手套，我这才发现，原来是我戴的那只防寒工作手套的虎口处开裂了，干活时虎口直接接触铁钩，冻伤了。

回复
Qizhen
2008年02月19日 08:58 编辑
是啊，我现在别提多怕过年了。

回复
yoyo
2008年02月18日 20:11 编辑
人到了一定年纪就会开始回忆 老兄 你已经开始了…

回复
acca
2008年02月18日 16:01 编辑
同理 南北极不要户外尿尿-.-

回复
sandy
2008年02月18日 09:10 编辑
好像记得你之前说过是东北哪里的，东北那边可真冷啊。

回复
